### PR TITLE
Export routes as middleware

### PR DESCRIPTION
**Pull request by michalholasek**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "http-zoo",
   "version": "0.1.0",
+  "main":"src/routes/index.js",
```
Comment (XVincentX) [inline package.json]: Nip: Add a space

File: package.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "http-zoo",
```
Comment (XVincentX) [inline package.json]: We should probably come up with a more meaningful name...I'd call it something like `node-hamms` to keep credit and honour to the initial guy who made it. What do you think?

**Review by XVincentX — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -1,6 +1,7 @@
...
   "version": "0.1.0",
```
Comment (XVincentX) [inline package.json]: I think `1.0.0` would be totally ok

File: package.json (hunk 2 above)
Comment (michalholasek) [reply package.json]: hamms is mentioned in the README (inspired by hamms), but sure, I have no problem with that.

**Pull request merged**